manage

```diff
@@ -275,13 +293,26 @@ elif [ "$1" == 'update' ];then
   cp -rf "${DIRECTORY}/update/pi-apps/apps/${app}" "${DIRECTORY}/apps/${app}"
   
   if [ "$installback" == 'yes' ];then
-    echo "$app was originally installed before updating. Reinstalling the new version now."
```

Comment:
Why was this line removed?

Reply:
There is a line above it mentioned `status "$app's install script has been updated. Reinstalling $app..."`.